gslab_scons/log.py

```diff
@@ -7,11 +7,13 @@
 import gslab_scons.misc as misc
 
 
-def start_log(mode, vers, log = 'sconstruct.log'):
+def start_log(mode, vers, cl_args_list = sys.argv, log = 'sconstruct.log'):
```

Comment:
Have you tested this usage outside of the template? I wonder how `sys.argv` is assigned and am worried that this is not sufficiently robust (e.g. `sys` must be imported each time).

Reply:
Works like a dream, since the logger scripts import sys. Let me know if you want to see more. 
<img width="534" alt="screen shot 2018-01-12 at 1 45 57 pm" src="https://user-images.githubusercontent.com/19935793/34896504-fa6b2996-f79e-11e7-8307-9ce42ad533f7.png">

Reply:
I'm good, go ahead. 🏒

Reply:
Can you approve the review please?